apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/jvm/gc/GCProvider.java

```diff
@@ -62,6 +62,9 @@ private GCMetricAccessor findByBeanName(String name) {
         } else if (name.equals("MarkSweepCompact")) {
             // Serial collector ( -XX:+UseSerialGC )
             return new SerialGCModule(beans);
+        } else if (name.indexOf("ZGC") > -1) {
+            // Serial collector ( -XX:+UseZGC )
```

Comment:
@xu1009 can you open a PR to correct this comment?

```suggestion
            // Z garbage collector ( -XX:+UseZGC )
```

Reply:
ok